tools/update_repo.sh

```diff
@@ -538,7 +538,7 @@ Origin: Tarantool
 Label: tarantool.org
 Suite: stable
 Codename: $loop_dist
-Architectures: amd64 source
+Architectures: amd64 arm64 source
```

Comment:
I see several other appearance of amd64 and x86_64, at least around packages removal, but also as the first `pack_rpm` parameter. Are you sure that everything works for all distributions that are added in this PR?

I looked at the `live/tmp` directory and I see only Debian Buster packages.

I looked at centos_7_arm and it fails:

> Error: No Package found for python3-gevent
> Error: No Package found for python3-pyyaml

Reply:
The first point still actual. From what I see now, I have doubts that it'll work correctly for all distros. Please, don't pass over testing of such changes. It is better to release one day later than find that we're released something that does not work for a user.

Reply:
(And don't expect that a reviewer will able to catch all problems just by glance on the code. Neither of us is author of this tool. Actually someone should test it and write here about results of testing. Normally it should be done by the author of changes.)

Reply:
I made attempt to rewrite the patch to follow the [idea](https://github.com/tarantool/tarantool/pull/6269#issuecomment-900711333) described below: just replace amd64/x86_64 with `${arch}`.